juniper-vpn.py

```diff
@@ -246,7 +246,18 @@ def action_key(self):
 
     def action_continue(self):
         # Yes, I want to terminate the existing connection
+
+        # Fix up the broken HTML
+        html = self.r.read()
+        html = html.replace('checkSelected()",', 'checkSelected()"')
```

Comment:
I guess you're fixing this line?

<td><input id="postfixSID_1" type="checkbox" onclick="checkSelected()",  name="postfixSID" value="053b26a4"/></td>

I can only find two checkSelected() references in the source, but would it be good to be more specific, in case the HTML changes ever? Or at least so it's clearer what's being fixed :)